src/schoology-extractor/edfi_schoology_extractor/api/__init__.py

```diff
@@ -3,4 +3,4 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
-__version__ = '0.1.0'
+__version__ = "0.1.0"
```

Comment:
Do you remember why we even have this setting here? I don't think we should be using it, as it doesn't actually mean anything.

Reply:
Agree.  I think it's from before we knew much about Python.

Reply:
Also, that edit was from an automated "black" reformat of the whole project via the command line.  The black VS Code integration isn't working for me right now either.